.travis/docs/all_content_index.rst

```diff
@@ -0,0 +1,13 @@
+Documentation Index
```

Comment:
I feel like these files aren't necessarily specific to Travis? Thoughts on maybe moving these to the top level docs directory?

Reply:
They are specific to the way we build docs for docs.pulpproject.org. You can build the docs in this repo by themselves and they will not include plugin information.